x/mongo/driver/topology/topology_test.go

```diff
@@ -84,6 +85,58 @@ func TestServerSelection(t *testing.T) {
 			t.Errorf("Incorrect sever selected. got %s; want %s", srvs[0].Addr, desc.Servers[0].Addr)
 		}
 	})
+	t.Run("Compatibility Error Min Version Too High", func(t *testing.T) {
+		topo, err := New()
+		noerr(t, err)
+		desc := description.Topology{
+			Kind: description.Single,
+			Servers: []description.Server{
+				{Addr: address.Address("one:27017"), Kind: description.Standalone, WireVersion: &description.VersionRange{Max: 11, Min: 11}},
+				{Addr: address.Address("two:27017"), Kind: description.Standalone, WireVersion: &description.VersionRange{Max: 9, Min: 2}},
+				{Addr: address.Address("three:27017"), Kind: description.Standalone, WireVersion: &description.VersionRange{Max: 9, Min: 2}},
+			},
+		}
+		serverDesc := description.Server{Addr: address.Address("two"), Kind: description.Standalone, WireVersion: &description.VersionRange{Max: 9, Min: 2}}
```

Comment:
Can we simplify by not going through `apply` at all? We already have spec tests for servers being/becoming incompatible, so we don't need to test the `apply` path here. Instead, we can do `t.desc.Store(desc)` and then try to select a server.

Reply:
sure, to get this test to work without calling apply, I've added added in `desc.compatibleErr`